.github/workflows/build.yml

```
@@ -168,24 +168,38 @@ jobs:
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
   test:
+    name: >-
+      test (${{ matrix.scenario }}) -
+      ${{ matrix.platform }}-${{ matrix.architecture }}
     needs:
       - diagnostics
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
+        architecture:
+          - amd64
+          # We cannot currently test _any_ ARM64 platforms under qemu
+          # because qemu cannot currently support iptables:
```

Nitpicking, but we should use all caps for QEMU in comments (see https://github.com/qemu/qemu).
```suggestion
          # We cannot currently test _any_ ARM64 platforms under QEMU
          # because QEMU cannot currently support iptables:
```